src/ghastoolkit/supplychain/dependencies.py

```diff
@@ -51,7 +51,7 @@ def getPurl(self, version: bool = True) -> str:
         if self.manager:
             result += f"{self.manager.lower()}/"
         if self.namespace:
-            result += f"{self.namespace}/"
+            result += f"{self.namespace}:"
```

Comment:
The `/` here is used to separate the namespace and name of the packages.

https://github.com/package-url/purl-spec?tab=readme-ov-file#purl

Is there a reason to change it?

Reply:
Yes, the formatting coming back from the perl lookup in policy as code returns it with : not a / so for me to be able to do proper comparisons there I made that change

Reply:
So if you look at, https://github.com/GeekMasher/ghastoolkit/pull/232/files#diff-5c97a427ff057c148668cb63e5aea09904fddb4c6494c8a2a1442d5f550266e7R97 that response is coming from the dependabot, the name of the package itself is being provided as x:y vs doing x/y so I had swapped to it for that as thats an upstream result, would you prefer I just do a find and replace?

Reply:
Thanks for tracking this down @Killklli, very strange its coming from Dependabot / DepGraph. Thank you for resolving this

Reply:
For now I have removed this small change as the rest of the code looks good. Lets talk offline to discuss this change more